docs/base_models/seggpt.md

```diff
@@ -0,0 +1,64 @@
+# SegGPT
+
+[SegGPT](https://github.com/baaivision/Painter/tree/main/SegGPT) is a transformer-based few-shot semantic segmentation model developed by [BAAI Vision](https://github.com/baaivision).
+
+This model performs well on task-specific segmentation tasks when given a few labeled images from which to learn features about the objects you want to identify.
+
+## Installation
+
+To use SegGPT with autodistill, you need to install the following dependency:
+
+
+```bash
+pip3 install autodistill-seggpt
+```
+
+## About SegGPT
+
+This Autodistill module uses a handful of pre-labelled images for improved accuracy.
+
+You will need some labeled images to use SegGPT. Don't have any labeled images? Check out [Roboflow Annotate](https://roboflow.com/annotate), a feature-rich annotation tool from which you can export data for use with Autodistill.
+
+## Quickstart
+
+```python
+from autodistill_seggpt import SegGPT,FewShotOntology
+
+base_model = SegGPT(
+    ontology=FewShotOntology(supervision_dataset)
+)
+
+base_model.label("./unlabelled-climbing-photos", extension=".jpg")
+```
+
+## How to load data from Roboflow
+
+Labelling and importing images is easy!
+
+You can use [Roboflow Annotate](https://roboflow.com/annotate) to label a few images (5-10 should work fine). For your Project Type, make sure to pick Instance Segmentation--you'll be labelling with polygons.
+
+Once you've labelled your images, you can press Generate > Generate New Version. You can use all the default options--no Augmentations are necessary.
+
+Once your dataset version is generated, you can press Export > Continue.
+
+Then you'll get some download code to copy--it should look something like this:
+
+```python
+!pip install roboflow
```

Comment:
Can you remove the `!pip install roboflow` statement? You should have this in a separate code snippet without the `!`. Users may not be using a notebook environment.

Reply:
I'd like to keep this as is--the default Roboflow Export code includes `!pip install roboflow`, so I think it'd be confusing to remove it from this example code snippet.

I added a comment below the code for non-notebook users--do you think this is alright?
> Note: if you are not using a notebook environment, you should remove `!pip install roboflow` from your code, and run `pip install roboflow` in your terminal instead.